coremltools/converters/keras/_layers2.py

```diff
@@ -880,7 +880,9 @@ def convert_simple_rnn(builder, layer, input_names, output_names, keras_layer):
     W_x = _np.zeros((hidden_size, input_size))
     b = None
 
-    if keras_layer.implementation == 0:
+    implementation = 0 if not hasattr(keras_layer, 'implementation') else \
```

Comment:
This is probably easier to understand:
`implementation = keras_layer.implementation if hasattr(keras_layer, 'implementation') else 0`

Reply:
Fixed